packages/table/stories/table-elements.stories.ts

```diff
@@ -53,7 +53,7 @@ export default {
 
 export const elements = (): TemplateResult => {
     return html`
-        <sp-table size="m">
+        <sp-table emphasized>
```

Comment:
again, I wonder if we want a majority of the examples to be emphasized, rather than the other way around, ie only specifying "emphasized" for a single story.

Reply:
I'm not sure which one we should be showing in most default examples. One thought to add to the discussion;

The default for table previously had the selected row in blue. With the addition of emphasized and non-emphasized, the default selected row color will be grey. So in one way, the previous default looked more like emphasized (the checkbox did not have emphasized colors though).

Reply:
That's interesting. I never realised that the checkbox and highlighted rows were different colours LOL 

I think I'd err on the side of matching our other component stories, which usually have a singular "emphasized" story and leave the rest as-is.